### DISCUSSION TITLE
How can I donate?

### DISCUSSION THREAD

**Discussion by PriceHiller** [williamboman/nvim-lsp-installer#424] [Q&A] [answered]
As far as I'm concerned, this project has saved me an awful lot of time mucking about setting up LSP servers... something that, frankly, is a means to an end. It isn't particularly fun in and of itself to set up LSP servers for me and while, yes, LSP config does provide simple snippets, it's extremely convenient to just install them on the fly. This project has gotten regular support and quick interactions in the issues, a rare thing, so I value what you've done quite a bit.

I'd love to send $50 a month or so your way considering how much time you've saved me, but I don't see a location to donate; how would I go about doing so?

**Comment by williamboman** [comment, marked as answer]
Oh thanks for your kind words! It makes me very happy to hear that it's made your life easier :). Maintaining this plugin has been very fun and I learn something new every day! I've never even considered donations/sponsorship as I think there are a ton of other projects worth supporting. I know GitHub has made this somewhat easy to set this up, I'll try to look into if it's even possible. Anyway, I recommend people to sponsor the neovim project instead, their efforts are what makes this all possible!

- https://github.com/sponsors/neovim

**Reply by PriceHiller** [reply]
If you're absolutely sure, I'll go throw some into the pot for neovim in general, but you really have saved me a substantial amount of pain. I think *your* project is one worth supporting, but if you really do prefer I send it elsewhere then I'll pop it into neovim's sponsor pool.

**Reply by williamboman** [reply]
Haha I'll leave it up to you! I enabled sponsorships for this project now, GitHub was really fast at approving it.
